.github/workflows/skywalking.yaml

```diff
@@ -52,30 +52,62 @@ jobs:
         with:
           submodules: true
       - name: Check code style
-        run: ./mvnw clean checkstyle:check
+        run: ./mvnw -B -q clean checkstyle:check
+
+  dependency-license:
+    if: |
+      always() &&
+      ((github.event_name == 'schedule' && github.repository == 'apache/skywalking') || needs.changes.outputs.pom == 'true')
```

Comment:
Does this skipping make this PR not changed?

Reply:
> Does this skipping make this PR not changed?

I don’t understand what you mean 😄

Reply:
GHA shows dependencies check skipped.